src/foundry/client/hooks.d.mts

```diff
@@ -810,18 +845,38 @@ declare global {
      */
     type CloseApplication<A extends Application = Application> = (app: A, html: JQuery) => boolean | void;
 
+    /** EffectsCanvasGroup */
+
+    /**
+     * A hook event that fires when a {@link CanvasGroup} is drawn.
+     * The dispatched event name replaces "Group" with the named CanvasGroup subclass, i.e. "drawPrimaryCanvasGroup".
+     * @param group - The group being drawn
+     */
+    type DrawGroup<
+      G extends ReturnType<typeof CanvasGroupMixin> = ReturnType<typeof CanvasGroupMixin<CanvasGroupMixin.BaseClass>>,
```

Comment:
Maybe this suggests we should have an _interface_ and possible namespace named `CanvasGroup` available in the global scope.

I kind of like the idea of being able to write `CanvasGroup.Any`.

On that note I'm pretty sure `ReturnType<typeof CanvasGroupMixin>` is the same thing as `ReturnType<typeof CanvasGroupMixin<CanvasGroupMixin.BaseClass>>` because when generics are left off they seem to default to their generic constraint.

So I think this could be `CanvasGroup.Any = CanvasGroup.Any` or possibly `CanvasGroupMixin.Any`.

Reply:
I'll write that up as a ticket?